Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -498,6 +498,7 @@
     - ChemMasterMachineCircuitboard
     - CondenserMachineCircuitBoard
     - HotplateMachineCircuitboard
+    - PrinterDocMachineCircuitboard # Corvax-Printer
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)